### PR TITLE
Akegalj/cse175 launch script

### PR DESCRIPTION
**Pull request by akegalj**
Adds launch script for staging

Tested and it works

### CODE REVIEW

File: frontend/scripts/build-frontend-simple.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+if ! which purescript-derive-lenses >/dev/null; then
+    echo "Installing purescript-derive-lenses..."
+    rm -rf purescript-derive-lenses
+    git clone https://github.com/paf31/purescript-derive-lenses
+    cd purescript-derive-lenses
+    git checkout 02457e6 # TODO: I am not sure which version should we use? @jens?
```
Comment (sectore) [inline frontend/scripts/build-frontend-simple.sh]: That's the correct commit of `purescript-derive-lenses` (see https://github.com/input-output-hk/cardano-sl-explorer/blob/cardano-sl-0.5-rc/frontend/purescript-derive-lenses.nix#L10)

**Comment by akegalj** [comment]
@sectore 
> Why do we need build-frontend-simple.sh? We already have build.sh https://github.com/input-output-hk/cardano-sl-explorer/blob/cardano-sl-0.5-rc/frontend/scripts/build.sh , which does almost the same.

yes, i am aware - but unfortunately I didn't have time to properly test it myself. I have added the script mainly to prepare testing for CSE-161 . In general I would remove it if it is not needed. 

In fact we will also remove these new staging scripts as we will merge into cardano-sl project that already have similar scripts.

In summary, script was added because it was tested and it worked on my machine - and I didn't have time to properly test other scripts. If you think its not needed then we can remove it and update README with the steps.

**Comment by akegalj** [comment]
@sectore lets not merge until we check your scripts. Thanks for bringing that up

**Comment by akegalj** [comment]
Merging because it would be helpful for CSE-161. These scripts will be removed soon as we merge explorer into cardano-sl

**Pull request merged**